<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>密聊测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.8.3"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { white-space: pre-wrap;word-wrap: break-word; }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 2ex;
    margin-bottom: 3ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    /* width: 500px;*/
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 99%;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #74A474; }
.failClass  { background-color: #FDD283; }
.errorClass { background-color: #FF6600; }
.passCase   { color: #6c6; }
.failCase   { color: #FF6600; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}

#div_base {
            position:absolute;
            top:0%;
            left:5%;
            right:5%;
            width: auto;
            height: auto;
            margin: -15px 0 0 0;
}
</style>

    <link href="http://cdn.bootcss.com/bootstrap/3.3.0/css/bootstrap.min.css" rel="stylesheet">
</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>
<div id="div_base">

<div class='page-header'>
<h1>密聊测试报告</h1>
<p class='attribute'><strong>开始时间:</strong> 2017-09-01 16:39:47</p>
<p class='attribute'><strong>运行时长:</strong> 0:00:55.534975</p>
<p class='attribute'><strong>状态:</strong> 失败 3 错误 2</p>

</div>
<p class='description'>test1: 点击红鲤鱼进行密聊发照片test2:发送密聊文本信息test3：发送表情test4:全局搜索密聊</p>



<div class="btn-group btn-group-sm">
<button class="btn btn-default" onclick='javascript:showCase(0)'>总结</button>
<button class="btn btn-default" onclick='javascript:showCase(1)'>失败</button>
<button class="btn btn-default" onclick='javascript:showCase(2)'>全部</button>
</div>
<p></p>
<table id='result_table' class="table table-bordered">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>测试套件/测试用例</td>
    <td>总数</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>查看</td>
</tr>

<tr class='errorClass'>
    <td>zuobiaoCase.Miliao_Model.Miliao</td>
    <td>4</td>
    <td>0</td>
    <td>3</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c1',4)">详情</a></td>
</tr>

<tr id='ft1.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_1</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        错误</a>

    <div id='div_ft1.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.1: Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 42, in test_1
    LoginPage().zhanghao().send_keys("13691515688")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webelement.py", line 347, in send_keys
    self._execute(Command.SEND_KEYS_TO_ELEMENT, {'value': keys_to_typing(value)})
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webelement.py", line 494, in _execute
    return self._parent.execute(command, params)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: UiAUtomator shut down unexpectedly

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 42, in test_1
    LoginPage().zhanghao().send_keys("13691515688")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webelement.py", line 347, in send_keys
    self._execute(Command.SEND_KEYS_TO_ELEMENT, {'value': keys_to_typing(value)})
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webelement.py", line 494, in _execute
    return self._parent.execute(command, params)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: UiAUtomator shut down unexpectedly



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.2' class='none'>
    <td class='failCase'><div class='testcase'>test_2</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.2')" >
        失败</a>

    <div id='div_ft1.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.2: Traceback (most recent call last):
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 47, in Miliao_Shurukuang
    __Miliao_Shurukuang= self.driver.find_element_by_id("com.shixinyun.zuobiao:id/chat_message_et")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 269, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 99, in test_2
    Miliao_Page().Miliao_Shurukuang().send_keys('测试密聊')
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 50, in Miliao_Shurukuang
    "定位密聊文本输入框"
AssertionError: 定位密聊文本输入框
Traceback (most recent call last):
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 47, in Miliao_Shurukuang
    __Miliao_Shurukuang= self.driver.find_element_by_id("com.shixinyun.zuobiao:id/chat_message_et")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 269, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 99, in test_2
    Miliao_Page().Miliao_Shurukuang().send_keys('测试密聊')
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 50, in Miliao_Shurukuang
    "定位密聊文本输入框"
AssertionError: 定位密聊文本输入框


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.3' class='none'>
    <td class='failCase'><div class='testcase'>test_3</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.3')" >
        失败</a>

    <div id='div_ft1.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.3: Traceback (most recent call last):
  File "/Users/shixin/zuobiao/CommonFunction/Face.py", line 12, in biaoqing
    __biaoqing=self.driver.find_element_by_id("com.shixinyun.zuobiao:id/chat_face_btn")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 269, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 107, in test_3
    Face().biaoqing().click()
  File "/Users/shixin/zuobiao/CommonFunction/Face.py", line 15, in biaoqing
    "定位表情按钮失败"
AssertionError: 定位表情按钮失败
Traceback (most recent call last):
  File "/Users/shixin/zuobiao/CommonFunction/Face.py", line 12, in biaoqing
    __biaoqing=self.driver.find_element_by_id("com.shixinyun.zuobiao:id/chat_face_btn")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 269, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 107, in test_3
    Face().biaoqing().click()
  File "/Users/shixin/zuobiao/CommonFunction/Face.py", line 15, in biaoqing
    "定位表情按钮失败"
AssertionError: 定位表情按钮失败


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.4' class='none'>
    <td class='failCase'><div class='testcase'>test_4</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.4')" >
        失败</a>

    <div id='div_ft1.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.4: Traceback (most recent call last):
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 10, in Back
    __Back=self.driver.find_element_by_id("com.shixinyun.zuobiao:id/back")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 269, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 125, in test_4
    Miliao_Page().Back().click()
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 13, in Back
    "定位返回按钮失败"
AssertionError: 定位返回按钮失败
Traceback (most recent call last):
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 10, in Back
    __Back=self.driver.find_element_by_id("com.shixinyun.zuobiao:id/back")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 269, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 125, in test_4
    Miliao_Page().Back().click()
  File "/Users/shixin/zuobiao/CommonFunction/Miliao_Page.py", line 13, in Back
    "定位返回按钮失败"
AssertionError: 定位返回按钮失败


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>unittest.suite._ErrorHolder</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a href="javascript:showClassDetail('c2',1)">详情</a></td>
</tr>

<tr id='ft2.1' class='none'>
    <td class='errorCase'><div class='testcase'>Miliao)</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.1')" >
        错误</a>

    <div id='div_ft2.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.1: Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 33, in tearDownClass
    cls.driver.quit()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 520, in quit
    self.execute(Command.QUIT)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started

Traceback (most recent call last):
  File "/Users/shixin/zuobiao/zuobiaoCase/Miliao_Model.py", line 33, in tearDownClass
    cls.driver.quit()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 520, in quit
    self.execute(Command.QUIT)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Appium_Python_Client-0.24-py3.5.egg/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium-3.0.2-py3.5.egg/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: A session is either terminated or not started



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='total_row'>
    <td>总计</td>
    <td>5</td>
    <td>0</td>
    <td>3</td>
    <td>2</td>
    <td>&nbsp;</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>

</div>
</body>
</html>
